Lib/test/test_configparser.py

```diff
@@ -521,7 +528,6 @@ def test_default_case_sensitivity(self):
             cf.get(self.default_section, "Foo"), "Bar",
             "could not locate option, expecting case-insensitive defaults")
 
-    @unittest.skipIf(os.name == "nt", "TODO: RUSTPYTHON, universal newlines")
```

Comment:
probably need to add this again, it skipped the tests for windows (which are now failing)

Reply:
@DimitrisJim restored thanks